player.html

```diff
@@ -1,130 +1,154 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8" />
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
         <title>Encarta Video Player</title>
-        <link rel="stylesheet" href="player.css">
+        <link rel="stylesheet" type="text/css" href="player.css">
     </head>
 
     <body>
         <div class="topnav">
-            <button type="button"><a href="">Search</a></button>
             <input type="text" placeholder="Search...">
+            <button type="button"><a href="">Search</a></button>
         </div>
 
-        <div class="sidebar">
-            <h6>Suggested Videos...</h6>
-            <a href="">
-                <div class="suggest1">
-                <img src="images/IT Security.jpg" height="200" width="300" alt="IT Security">
-                <p class="suggest-head1"><b>Essentials of I. T. Security</b></p>
```

Comment:
Nice work. I will recommend you remove the <b> inline styling from your HTML and use styling only in CSS.